src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 spring.main.banner-mode=off
 server.port=8080
+#arm
```

Comment:
Please Write Full Name

Reply:
Code modified as per requirement. Could you merge.